pkg/node/state_fsm/fsm_sync.go

```diff
@@ -35,6 +35,20 @@ func (c conf) Now(tm types.Time) conf {
 type noopWrapper struct {
 }
 
+var (
+	syncSkipMessageList = proto.PeerMessageIDs{
+		proto.ContentIDGetBlock,
```

Comment:
GetBlock and GetBlockIds messages should be allowed to enable sync after Protobuf blocks activation.